Automattic/wp-calypso
# Stats: Utilize stats-list-legend in all components

Continuing to clean up redundant code in stats components by utilizing the new `StatsListLegend` component throughout all stats components.  The `StatsListLegend` renders the markup that produces these legends on each stats list:

<img width="733" alt="stats_ _wordpress_com_news_ _wordpress_com" src="https://cloud.githubusercontent.com/assets/22080/12513696/0afde482-c0d4-11e5-8373-ab75c9c6b0b6.png">

__To Test__
- View an insights page and verify all the legends look the same
- View a stats period page and verify they look good as well
